### PR TITLE
NIAD-575 Get SCR ID - OpenAPI spec

### PR DESCRIPTION
**Pull request by dawidskainos**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/summary-care-record.yaml [hunk 1]
```diff
@@ -154,14 +154,66 @@ paths:
...
+          name: _sort
+          description: Defines how Patient's SCR list should be sorted in order to retrieve the latest one. The only supported value is _sort=date
+          required: false
+          example: date
+          schema:
+            type: string
+             
+        - in: query
+          name: _count
+          description: Defines the number of latest patient SCR IDs that sgould be retrieved. Currently the only supported value is _count=1.
```
Comment (bartek-sarul) [inline specification/summary-care-record.yaml:202]: Will this throw 400 if other value is passed in?

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

File: specification/components/examples/DocumentReferenceBundle.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "Bundle",
+  "id": "55f8ba57-761a-406e-b61d-423027830fef",
+  "type": "searchset",
+  "total": 1,
+  "entry": [
+    {
+      "fullUrl": "localhost:9000/DocumentReference/6d334ca8-a6e1-4bae-93b1-19a14a36ea7a",
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/components/examples/DocumentReferenceBundle.json]: Fixed.

File: specification/components/examples/DocumentReferenceBundle.json [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          "system": "https://fhir.nhs.uk/Id/nhsSCRUUID",
+          "value": "FA60BE64-1F34-11EB-A2A8-000C29A364EB"
+        },
+        "status": "current",
+        "type": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "196981000000101",
+              "display": " General Practice Summary"
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/components/examples/DocumentReferenceBundle.json]: Fixed.

File: specification/components/examples/DocumentReferenceBundle.json [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+          }
+        ],
+        "context": {
+          "event": [
+            {
+              "coding": [
+                {
+                  "system": "http://snomed.info/sct",
+                  "code": "196981000000101",
+                  "display": " General Practice Summary"
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/components/examples/DocumentReferenceBundle.json]: Fixed.

File: specification/components/schemas/DocumentReference.yaml [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+          items:
+            type: object
+            properties:
+              event:
+                type: object
+                properties:
+                  coding:
+                    $ref: coding.yaml
+                  
+
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/components/schemas/DocumentReference.yaml]: Fixed.

File: specification/summary-care-record.yaml [hunk 6]
```diff
@@ -154,14 +154,66 @@ paths:
...
+          name: patient
+          description: The patient's NHS Number. Must be preceded with FHIR identifier (eg."patient=https://fhir.nhs.uk/Id/nhs-number|9995000180")
+          required: true
+          example: https://fhir.nhs.uk/Id/nhs-number|9995000180
+          schema:
+            type: string
+            
+        - in: query
+          name: type
+          description: General Practice Summary snomed type. Must be equal "type=http://snomed.info/sct|196981000000101"
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/summary-care-record.yaml]: Fixed.

File: specification/summary-care-record.yaml (hunk 1 above)
Comment (dawidskainos) [reply specification/summary-care-record.yaml]: Yes. I added some additional description so it's clear.

File: specification/summary-care-record.yaml [hunk 7]
```diff
@@ -154,14 +154,66 @@ paths:
...
+          required: false
+          example: http://snomed.info/sct|196981000000101
+          schema:
+            type: string
+          
+        - in: query
+          name: _sort
+          description: Defines how Patient's SCR list should be sorted in order to retrieve the latest one. The only supported value is _sort=date
```
[the remark this answers is not in the document]
Comment (dawidskainos) [reply specification/summary-care-record.yaml]: Yes. I added some additional description so it's clear.

**Review by bartek-sarul — APPROVED** [review]

**Pull request merged**